<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start/Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mindwave device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deviceComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>/dev/rfcomm0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/rfcomm1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/rfcomm2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyS0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyS1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyS2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/tty0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/tty1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/tty2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Eye blink sensitivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eyeBlinkSensitivity">
        <property name="text">
         <string>0.45</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lower Eye blink frequency  (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lowerEyeBlinkFrequency">
        <property name="text">
         <string>5.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Higher Eye blink frequency (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="higherEyeBlinkFrequency">
        <property name="text">
         <string>20.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Setup channel mapping&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="rescanMidiButton">
        <property name="text">
         <string>Rescan midi devices</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>MIDI chan 8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QComboBox" name="midiDevice2ComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>MIDI chan 0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QComboBox" name="midiDevice3ComboBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="midiDevice0ComboBox"/>
      </item>
      <item row="2" column="11">
       <widget class="QComboBox" name="midiDevice7ComboBox"/>
      </item>
      <item row="2" column="8">
       <widget class="QComboBox" name="midiDevice6ComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>MIDI chan 4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="midiDevice8ComboBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="midiDevice4ComboBox"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>MIDI chan 5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QComboBox" name="midiDevice5ComboBox"/>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>MIDI chan 14</string>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QComboBox" name="midiDevice14ComboBox"/>
      </item>
      <item row="4" column="9">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>MIDI chan 15</string>
        </property>
       </widget>
      </item>
      <item row="4" column="11">
       <widget class="QComboBox" name="midiDevice15ComboBox"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>MIDI chan 9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QComboBox" name="midiDevice9ComboBox"/>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>MIDI chan 10</string>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QComboBox" name="midiDevice10ComboBox"/>
      </item>
      <item row="3" column="9">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>MIDI chan 11</string>
        </property>
       </widget>
      </item>
      <item row="3" column="11">
       <widget class="QComboBox" name="midiDevice11ComboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>MIDI chan 12</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="midiDevice12ComboBox"/>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>MIDI chan 13</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QComboBox" name="midiDevice13ComboBox"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>MIDI chan 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>MIDI chan 7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>MIDI chan 3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>MIDI chan 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="midiDevice1ComboBox"/>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>MIDI chan 6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="resetChannel4Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="resetChannel8Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="resetChannel12Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="resetChannel0Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="resetChannel1Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="resetChannel5Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="resetChannel9Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="resetChannel13Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="resetChannel2Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="resetChannel6Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="resetChannel10Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="resetChannel14Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QPushButton" name="resetChannel3Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QPushButton" name="resetChannel7Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="3" column="10">
       <widget class="QPushButton" name="resetChannel11Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="4" column="10">
       <widget class="QPushButton" name="resetChannel15Button">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Setup event mapping&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QWidget" name="meditationPluginWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="meditationCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Meditation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="meditationPluginComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="eyeBlinkCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Eye blink</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="eyeBlinkPluginComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QWidget" name="eyeBlinkPluginWidget" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="attentionPluginWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="attentionCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Attention</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="attentionPluginComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuState">
    <property name="title">
     <string>State</string>
    </property>
    <addaction name="actionLoad_state"/>
    <addaction name="actionSave_state"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuState"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionLoad_state">
   <property name="text">
    <string>Load state</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>deviceComboBox</tabstop>
  <tabstop>eyeBlinkSensitivity</tabstop>
  <tabstop>lowerEyeBlinkFrequency</tabstop>
  <tabstop>higherEyeBlinkFrequency</tabstop>
  <tabstop>rescanMidiButton</tabstop>
  <tabstop>midiDevice0ComboBox</tabstop>
  <tabstop>midiDevice1ComboBox</tabstop>
  <tabstop>midiDevice2ComboBox</tabstop>
  <tabstop>midiDevice3ComboBox</tabstop>
  <tabstop>midiDevice4ComboBox</tabstop>
  <tabstop>midiDevice5ComboBox</tabstop>
  <tabstop>midiDevice6ComboBox</tabstop>
  <tabstop>midiDevice7ComboBox</tabstop>
  <tabstop>midiDevice8ComboBox</tabstop>
  <tabstop>midiDevice9ComboBox</tabstop>
  <tabstop>midiDevice10ComboBox</tabstop>
  <tabstop>midiDevice11ComboBox</tabstop>
  <tabstop>midiDevice12ComboBox</tabstop>
  <tabstop>midiDevice13ComboBox</tabstop>
  <tabstop>midiDevice14ComboBox</tabstop>
  <tabstop>midiDevice15ComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
